server/web_templates.go

```diff
@@ -51,7 +56,7 @@ var indexTemplate = template.Must(template.New("index.html.tmpl").Parse(`
 <div class="container">
   <section class="header">
     <a title="atlantis" href="/"><img src="/static/images/atlantis-icon.png"/></a>
-    <p class="title-heading">atlantis</p>
+    <p class="title-heading">atlantis {{ .Version }}</p>
```

Comment:
I think this makes more sense to be in the bottom left of footer with the prefix `v`. I don't like having it in the title here because I think it gives the version too much prominence. If you don't want to mess with the CSS for a fixed footer that's totally fine, just LMK and I can merge your PR and then add that myself.